virl2_client/models/lab.py

```diff
@@ -617,7 +658,7 @@ def remove_node(self, node: Node | str, wait: bool | None = None) -> None:
         """
         Remove a node from the lab.
 
-        If you have a node object, you can also simply do::
```

Comment:
I don't think this was a typo.

Reply:
I will revert it then, after you finish review.